integration-tests/initiateDataRequest.spec.ts

```diff
@@ -4,72 +4,112 @@ import { approveZendeskTicket } from './utils/zendesk/approveZendeskTicket'
 import { deleteZendeskTicket } from './utils/zendesk/deleteZendeskTicket'
 import { invalidRequestData, validRequestData } from './constants/requestData'
 import {
+  ANALYSIS_BUCKET_NAME,
+  AUDIT_BUCKET_NAME,
   INITIATE_DATA_REQUEST_LAMBDA_LOG_GROUP,
-  PROCESS_DATA_REQUEST_LAMBDA_LOG_GROUP
+  INTEGRATION_TEST_DATE_PREFIX,
+  PROCESS_DATA_REQUEST_LAMBDA_LOG_GROUP,
+  TEST_FILE_NAME
 } from './constants/awsParameters'
 import { FilteredLogEvent } from '@aws-sdk/client-cloudwatch-logs'
 import { getZendeskTicket } from './utils/zendesk/getZendeskTicket'
 import { listZendeskTicketComments } from './utils/zendesk/listZendeskTicketComments'
+import { deleteAuditData } from './utils/aws/s3DeleteAuditData'
+import { copyAuditDataFromTestDataBucket } from './utils/aws/s3CopyAuditDataFromTestDataBucket'
 
 describe('Submit a PII request with approved ticket data', () => {
-  jest.setTimeout(60000)
+  jest.setTimeout(300000)
```

Comment:
Can this test actually take 5 minutes?  Is it possible to enter a lower number here incase something hangs?

Reply:
Decreased to 90000